### DISCUSSION TITLE
Properties varying Temperature and Pressure

### DISCUSSION THREAD

**Discussion by filipexc** [DanWBR/dwsim#670] [Q&A] [answered]
Hi Daniel,

I’ve attached the simulation we discussed earlier. I’m trying to determine the properties of a mixture at specific temperature and pressure conditions. However, we’re encountering an issue where, at certain (T,P) points as in the image bellow (P= 68bar T=40ºC), there’s a sudden phase change from fvap=1 to fvap=0, and then it reverts back to fvap=1, even when far from the critical point.

We would greatly appreciate your help in this case.
![image](https://github.com/user-attachments/assets/ad54f33c-42b8-47fa-b17b-f8f6e5a0456a)
[Properties_415.xlsx](https://github.com/user-attachments/files/16787600/Properties_415.xlsx)

https://drive.google.com/drive/folders/1fvGiD-rmAF92v7rYkSJyoNvXxmZOmkrT?usp=sharing (attached the file to drive, github wasn't allowing upload

**Comment by DanWBR** [comment, marked as answer]
this is a very difficult system (CH4, CO2, H2O...), very difficult to calculate the critical point and may also have two liquid phases depending on the P/T conditions.

![image](https://github.com/user-attachments/assets/1c2e04a5-7054-4568-8abb-b368ae818776)
![image](https://github.com/user-attachments/assets/1607ef41-7575-41dd-97b3-2e1178778b7f)

I recommend using GERG-2008 or PR/SRK with these settings:

![image](https://github.com/user-attachments/assets/eac987fd-d917-4eda-b08f-001289655cd6)
